tajo-plan/src/main/java/org/apache/tajo/plan/LogicalPlanner.java

```diff
@@ -485,9 +485,9 @@ public static void verifyProjectedFields(QueryBlock block, Projectable projectab
 
         for (int i = 0; i < groupingKeyNum; i++) {
           Target target = groupbyNode.getTargets()[i];
-          if (groupbyNode.getTargets()[i].getEvalTree().getType() == EvalType.FIELD) {
+          if (target.getEvalTree().getType() == EvalType.FIELD) {
             FieldEval grpKeyEvalNode = target.getEvalTree();
-            if (!groupbyNode.getInSchema().contains(grpKeyEvalNode.getColumnRef())) {
+            if (!groupbyNode.getInSchema().containsByQualifiedName(grpKeyEvalNode.getColumnRef().getQualifiedName())) {
```

Comment:
Schema.contains() internally calls the same logic with containsByQualifiedName(). Would you explain why you changed this line?